### ISSUE TITLE
Usability issue in some rules

### ISSUE THREAD

**Issue by annethyme** [act-rules/act-rules.github.io#613]
When going through a few ACT rules with our accessibility testing team, it turned out that some rules make most sense to those who wrote them, and little to everyone else. 
While many of our rules are now very precise and leave little room for (mis)interpretation, it can be hard to tell what the rule actually tests for, put in normal language.

It would be a shame, if different organisations end up having their own "understanding documents" for ACT Rules. 
It would be much better, if we could provide easier to understand guidance within the rules themselves, if/when needed.

One solution we discussed is to use notes to provide a "summary" of e.g. the applicability or expectation, when it can be hard to understand, for example: 

[Document title is descriptive](https://act-rules.github.io/rules/c4a8a4)

> ## Applicability
> 
> This rule applies to the first HTML `title` element that
> 
> - is a [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) of a [document element](https://www.w3.org/TR/dom/#document-element) that is an HTML `html` element, and
> - contains [children](https://www.w3.org/TR/dom/#concept-tree-child) that are [text nodes](https://www.w3.org/TR/dom/#text) that are not only [whitespace](#whitespace).
> 
> [NEW] **Note:** In essence, this rule applies to the `title` element of the document, if it contains text.

Questions for the CG: 
- Do others feel that this is an issue?, and if yes:
- What do others think about using notes as a solution for this? 
- Are there other ideas for how to solve this? 
- And how do we get a solution into existing and new rules?

**Comment by nitedog** [comment]
If we want to provide such summaries then I suggest we use something other than "Note" to identify them, to avoid confusion with other types of notes. I also suggest making such summaries more succinct.

For example:

> **Summary:** This rule applies to non-empty `title` elements of HTML pages.

**Comment by corbb** [comment]
> If we want to provide such summaries then I suggest we use something other than "Note" to identify them, to avoid confusion with other types of notes. I also suggest making such summaries more succinct.
* I'm fine calling these "notes" whatever makes sense. "Plain-language Definition" would be fine.
* With all due respect, the readability problem for my team of accessibility testers comes from the desire of rule authors to be succinct. While specificity is important for standardization, that level of specificity comes at a cost of readability.
> For example:
> 
> > **Summary:** This rule applies to non-empty `title` elements of HTML pages.
"non-empty" is clear to several of us who have been reading WCAG or other specs for some time. I don't want to dwell on one rule here, so I'll reserve comments about the recommended solution for a future thread.

**Comment by EmmaJP** [comment]
I would hope that most ACT Rules are clear enough not to need notes or summaries, as helping testing teams to share a common understanding of WCAG was one of the original aims.

However, Anne's example note acts as a nice plain english synopsis of what has just been more precisely specified. I'm not sure it needs to have "Note" or "Summary" at the start. An "In essence..." sentence at the end of a more complex applicability explanation could be sufficient, and would retain its context. ie:

```
**Applicability**
This rule applies to the first HTML title element that:

- is a descendant of a document element that is an HTML html element, and
- contains children that are text nodes that are not only whitespace.

In essence, this rule applies to the title element of an HTML document, if it contains text.
```

**Comment by carlosapaduarte** [comment]
The ACT Rules Format specifies that applicability and expectation should be "written in plain language". I guess most of the rules don't' adhere to this format. I understand that writing this kind of information unambiguously in plain language could be impossible. 

I think @annethyme suggestion might be a workaround but we probably need a plain language "introduction" that is complemented by the unambiguous specification. However, I think we already have it. For the rule used as an example, the Description is "This rule checks that the document title describes the topic or purpose of the document." What the suggested note adds to this is just that tile element contains text. Can't we just add that to the description?

**Comment by Brynanders** [comment]
Agree with Carlos, my understanding was that the "rule description" summarises, in plain language, what the rule is doing.

**Comment by annethyme** [comment]
Seems there is agreement that if the Description is not already providing an easy to understand explanation of what the rule is testing, it should be improved to do so. 

If we find rules where the Description is not enough, I suggest we raise an issue to improve that particular Description.
